packages/kotti-popover/src/Popover.vue

```diff
@@ -49,6 +50,7 @@ export default {
 	},
 	methods: {
 		handleClick() {
+			if (this.disabled) return null
```

Comment:
```suggestion
			if (this.disabled) return
```

Alternatively, use `if (!this.disabled) this.showPopper = !this.showPopper`.